pipeline/transformer/transformer_miro/src/main/scala/uk/ac/wellcome/platform/transformer/miro/transformers/MiroImage.scala

```diff
@@ -0,0 +1,41 @@
+package uk.ac.wellcome.platform.transformer.miro.transformers
+
+import uk.ac.wellcome.models.work.internal.{
+  DigitalLocation,
+  IdentifierType,
+  LocationType,
+  SourceIdentifier,
+  UnmergedImage,
+  Unminted
+}
+import uk.ac.wellcome.platform.transformer.miro.source.MiroRecord
+
+trait MiroImage {
```

Comment:
Personally don't like the mixin pattern in scala, as can lead to cases where we extend from something we no longer actually use, and the compiler doesn't catch it (lots of our tests suffer for this). If a trait is just a collection of functions, and not a contract to be implemented, I would generally just pull them out into an `object` or something.

Reply:
(also harder for people who don't use an IDE to understand where a function is coming from :wink:)

Reply:
I agree and I'll change `MetsDataImageUtils` accordingly, but here in `MiroImage` I was copying the existing convention where it's all traits:
![image](https://user-images.githubusercontent.com/4429247/74249126-90909080-4ce0-11ea-99ef-1cb48722aae5.png)

I would lean towards making the minimal change but the other options would be to make this an odd-one-out object, or changing all of them to be objects etc  - what do you think?

Reply:
Ah yep, I reckon just make the minimal change in that case

Reply:
(I mean follow the existing convention of using a trait, not having a one off object)

Reply:
Convention might have happened because others were just following.... 🚶‍♀ 🚶 🚎

Reply:
> Convention might have happened because others were just following.... :walking_woman: :walking_man: :trolleybus:

 :yin_yang:  :open_mouth: :yin_yang: